src/main/java/ch/njol/skript/util/DelayedChangeBlock.java

```diff
@@ -1,375 +1,381 @@
+	}
+	
+	@Override
+	public int getBlockPower(final BlockFace face) {
+		return b.getBlockPower(face);
+	}
+	
+	@Override
+	public int getBlockPower() {
+		return b.getBlockPower();
+	}
+	
+	@Override
+	public boolean isEmpty() {
+		Material type = getType();
+		assert type != null;
+		return BlockCompat.INSTANCE.isEmpty(type);
+	}
+	
+	@Override
+	public boolean isLiquid() {
+		Material type = getType();
+		assert type != null;
+		return BlockCompat.INSTANCE.isLiquid(type);
+	}
+	
+	@Override
+	public double getTemperature() {
+		return b.getTemperature();
+	}
+	
+	@Override
+	public double getHumidity() {
+		return b.getHumidity();
+	}
+	
+	@Override
+	public PistonMoveReaction getPistonMoveReaction() {
+		return b.getPistonMoveReaction();
+	}
+	
+	@Override
+	public boolean breakNaturally() {
+		if (newState != null) {
+			return false;
+		} else {
+			Bukkit.getScheduler().scheduleSyncDelayedTask(Skript.getInstance(), new Runnable() {
+				@Override
+				public void run() {
+					b.breakNaturally();
+				}
+			});
+			return true;
+		}
+	}
+	
+	@Override
+	public boolean breakNaturally(final ItemStack tool) {
+		if (newState != null) {
+			return false;
+		} else {
+			Bukkit.getScheduler().scheduleSyncDelayedTask(Skript.getInstance(), new Runnable() {
+				@Override
+				public void run() {
+					b.breakNaturally(tool);
+				}
+			});
+			return true;
+		}
+	}
+	
+	@Override
+	public Collection<ItemStack> getDrops() {
+		return b.getDrops();
+	}
+	
+	@Override
+	public Collection<ItemStack> getDrops(final ItemStack tool) {
+		return b.getDrops(tool);
+	}
+	
+	@Nullable
+	@Override
+	public Location getLocation(final @Nullable Location loc) {
+		if (loc != null) {
+			loc.setWorld(getWorld());
+			loc.setX(getX());
+			loc.setY(getY());
+			loc.setZ(getZ());
+			loc.setPitch(0);
+			loc.setYaw(0);
+		}
+		return loc;
+	}
+
+	@Override
+	public void setType(Material type, boolean applyPhysics) {
+		if (newState != null) {
+			newState.setType(type);
+		} else {
+			Bukkit.getScheduler().scheduleSyncDelayedTask(Skript.getInstance(), new Runnable() {
+				@Override
+				public void run() {
+					b.setType(type, applyPhysics);
+				}
+			});
+		}
+	}
+
+	@Override
+	public BlockData getBlockData() {
+		return b.getBlockData();
+	}
+
+	@Override
+	public void setBlockData(BlockData data) {
+		setBlockData(data, true);
+	}
+
+	@Override
+	public void setBlockData(BlockData data, boolean applyPhysics) {
+		if (newState != null) {
+			newState.setBlockData(data);
+		} else {
+			b.setBlockData(data, applyPhysics);
+		}
+	}
+	
+	@Nullable
+	@Override
+	public RayTraceResult rayTrace(Location start, Vector direction, double maxDistance, FluidCollisionMode fluidCollisionMode) {
+		return b.rayTrace(start, direction, maxDistance, fluidCollisionMode);
+	}
+	
+	@Override
+	public boolean isPassable() {
+		return isPassable;
+	}
+
+	@Override
+	public BoundingBox getBoundingBox() {
+		return b.getBoundingBox();
+	}
+	
+	@Override
```

Comment:
Configure your editor to use LF instead of CRLF and to not replace them, it'll cause this kind of diff otherwise.

Reply:
IntelliJ is already set up to use LF. Im not exactly sure why this happened in this class and not the other.